content/en/dashboards/template_variables.md

```diff
@@ -58,45 +58,41 @@ On log, APM, and RUM widgets, you can use wildcards in the middle of a value (fo
-To save the current values of the template variables in a view, click on the dropdown menu and click *Save selections as view*. From there, you are prompted to enter a unique name for the view. After saving, this view is listed in the dropdown menu. Click on this view to retrieve the previously saved values for the template variables.
+To save the current values of the template variables in a view, click on the *Saved Views* dropdown menu and click *Save selections as view*. From there, you are prompted to enter a unique name for the view. After saving, this view is listed in the dropdown menu. Click on this view to retrieve the previously saved values for the template variables.
 
 #### Delete
 
 To delete a view, click on the saved views dropdown and choose *Manage views...*. From there, a popup with your saved views is displayed with a trash bin icon next to each view. Click the appropriate trash bin icon to delete a view.
 
-{{< img src="dashboards/template_variables/manage_views.png" alt="Manage View Popup" style="width:75%;">}}
-
 #### Modify
 
-To modify the *(Default Value)* view, click on the pencil icon and update the template variable values. Then click *Done* to save. If any values in the other views are changed, save the values as a new view, and then delete the original view.
+To modify the *Default view*, click on the pencil icon and update the template variable values. Then click *Done* to save. If any values in the other views are changed, save the values as a new view, and then delete the original view.
 
 ## Use
 
 Template variables are used in widgets and event overlays.
 
 ### Widgets
 
-When creating or editing a widget, existing template variables display as options in the `from` field. For example, if you create the template variable `env`, the option `$env` is available.
-
-After the widget is saved, the value of the template variable is the one selected from the top of your dashboard:
+When creating or editing a widget, existing template variables display as options in the `from` field. For example, if you configure the template variable `environment`, the option `$environment` is available as a dynamic variable in the widget. 
 
-{{< img src="dashboards/template_variables/selecting_template_variables.png" alt="Selecting template variables" style="width:75%;">}}
+{{< img src="dashboards/template_variables/dynamic_template_variable.png" alt="Template variable can be set dynamically in widgets" style="width:100%;">}}
 
-To query based on just a prefix or suffix, use a wildcard character (`*`) at the beginning or end of the template variable value. For example, find all regions that begin with `us` using `us*`, or all emails under the `example.com` domain with `*@example.com`.
+Selecting the `environment` value of "production", dynamically scopes widgets with the `$environment` variable to the production environment.
```

Comment:
```suggestion
Selecting **production** for the `environment` value dynamically scopes widgets with the `$environment` variable to the production environment.
```

Reply:
I wasn't sure how to let the customer know that "production" is an example value. Would an italic work better here?